source/docs/software/advanced-controls/geometry/coordinate-systems.rst

```diff
@@ -0,0 +1,24 @@
+Coordinate Systems
+==================
+
+In FRC, there are two main coordinate systems that we use for representing objects' positions.
+
+Field Coordinate System
+-----------------------
+
+The field coordinate system (or global coordinate system) is an absolute coordinate system where a point on the field is designated as the origin. Positive :math:`theta` is in the counter-clockwise direction, and the positive x-axis points away from your alliance's driver station wall, and the positive y-axis is perpendicular and to the left of the positive x-axis.
+
+.. image:: diagrams/coordinate-systems-1.drawio.svg
+
+Below is an example of a field coordinate system overlayed on the 2020 FRC field.
```

Comment:
You need to specify that this system is for the red alliance. Maybe it would be helpful to show red and blue systems on the image.